keras_core/ops/numpy.py

```diff
@@ -3457,3 +3457,21 @@ def compute_output_spec(self, N, M=None, k=0, dtype="float32"):
 @keras_core_export(["keras_core.ops.eye", "keras_core.ops.numpy.eye"])
 def eye(N, M=None, k=0, dtype="float32"):
     return backend.numpy.eye(N, M=M, k=k, dtype=dtype)
+
+
+class FloorDiv(Operation):
+    def call(self, x1, x2):
+        return backend.numpy.floordiv(x1, x2)
+
+    def compute_output_spec(self, x1, x2):
+        x1_shape = getattr(x1, "shape", [])
+        x2_shape = getattr(x2, "shape", [])
+        output_shape = broadcast_shapes(x1_shape, x2_shape)
+        return KerasTensor(output_shape, dtype=x1.dtype)
+
+
+@keras_core_export(["keras_core.ops.floor", "keras_core.ops.numpy.floor"])
+def floordiv(x1, x2):
```

Comment:
If it's exported as `floor` then it should be called `floor` and the op class should be `Floor`.

Reply:
Changed export to floordiv